fe/fe-core/src/main/cup/sql_parser.cup

```diff
@@ -253,7 +253,7 @@ terminal String KW_ADD, KW_AFTER, KW_AGGREGATE, KW_ALL, KW_ALTER, KW_AND, KW_ARR
     KW_PLUGIN, KW_PLUGINS,
     KW_PROC, KW_PROPERTIES, KW_PROPERTY, KW_PERCENTILE, KW_PERCENTILE_UNION,
     KW_QUERY, KW_QUOTA,
-    KW_RANDOM, KW_RANGE, KW_READ, KW_RECOVER, KW_REGEXP, KW_RELEASE, KW_RENAME,
```

Comment:
We can make `PARTITION BY RANDOM` work,  when users don't specify `DISTRIBUTED BY` clause, then we use `RANDOM` strategy.

Reply:
KW_RANDOM is useless